src/SwedbankPay.Sdk.Tests/PaymentOrderTests.cs

```diff
@@ -97,7 +85,7 @@ public async Task CreateAndUpdatePaymentOrder_ShouldReturnPaymentOrderWithNewAmo
             var updateRequest = new UpdateRequest(Amount.FromDecimal(newAmount), Amount.FromDecimal(newVatAmount));
             Assert.NotNull(paymentOrder.Operations.Update);
 
-            var response = await paymentOrder.Operations.Update?.Invoke(updateRequest);
+            var response = await paymentOrder.Operations.Update(updateRequest);
```

Comment:
Update could be null here. So that's why ?.Invoke() was used. 
There should also be an Assert.NotNull(response) in case it would be null.

Reply:
If it can be null then the `Assert.NotNull(paymentOrder.Operations.Update)` two lines above should fail.

Reply:
True